src/App.js

```diff
@@ -15,7 +17,7 @@ function App() {
       <Routes>
         <Route path='/' element={<Home />} />
         <Route path='/reviews' element={<AllReviews />} />
-
+        <Route path='/reviews/:category' element={<ReviewsByCategory />} />
```

Comment:
Do we really need a different component called ReviewsByCategory? This is doing what AllReviews is doing. You could just put some conditional logic in AllReviews so it filters reviews by category :)